src/raftstore/store/store.rs

```diff
@@ -1264,6 +1266,8 @@ impl<T: Transport, C: PdClient> Store<T, C> {
                 continue;
             }
 
+            total_gc_logs += compact_idx - first_idx;
```

Comment:
Here we just want to know how many raft logs the leader peer tries to GC, most of the case, it is accurate. 

If in raft log gc worker, we have to distinguish the leader and the follower.